src/base/opt/mod.rs

```diff
@@ -565,6 +565,14 @@ impl<Octs> OptRecord<Octs> {
         self.flags & 0x8000 != 0
```

Comment:
It might be advisable to extract this repeated use of 0x8000 to a constant to avoid an accidental change later to one occurence but not the others.

Reply:
Can you take this up with Martin? It is his choice to not have a constant for the DO bit.

Reply:
As it was already discussed let's just leave it for now. I'm not a fan of repeated use of identical values as in my experience they are a foot gun waiting to happen, but a named constant would be less readable than the hex manipulation as the latter is very common in working with bit flags.